src/Microsoft.ML.FastTree/FastTreeClassification.cs

```diff
@@ -223,18 +223,18 @@ private IEnumerable<bool> GetClassificationLabelsFromRatings(Dataset set)
             return set.Ratings.Select(x => x >= 1);
         }
 
-        protected override void PrepareLabels(IChannel ch)
+        private protected override void PrepareLabels(IChannel ch)
         {
             _trainSetLabels = GetClassificationLabelsFromRatings(TrainSet).ToArray(TrainSet.NumDocs);
             //Here we set regression labels to what is in bin file if the values were not overriden with floats
```

Comment:
>//Here we set regression labels to what is in bin file if the values were not overriden with floats [](start = 12, length = 99)

Not related to your change, but - should this be deleted?

Reply:
It may be, I might prefer to wait on that for now.